portal/views/assessment_engine.py

```diff
@@ -774,8 +778,9 @@ def get_assessments():
 
 @assessment_engine_api.route(
     '/patient/<int:patient_id>/assessment',
-    methods=('PUT',),
+    methods=('OPTIONS', 'PUT')
 )
+@crossdomain(origin='*', headers=('Content-Type', 'Authorization'))
```

Comment:
Annoyed to find that the JSON `Content-Type` isn't listed here :/ (and thus why we need to add it to `Access-Control-Allow-Headers`)
https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
```
The only allowed values for the Content-Type header are:
application/x-www-form-urlencoded
multipart/form-data
text/plain
```